graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -25,6 +25,7 @@
 from time import perf_counter
 from typing import Any, Dict, Optional, Set, Tuple
 
+from pyspark import RDD
```

Comment:
Where do we use RDD here?

Reply:
It's used in a type annotation within the file.